IndyAV/localization/README.md

```diff
@@ -0,0 +1,3 @@
+# Localization
```

Comment:
Talk about the sylphase and 3d lidar mapping that we already have demonstrated and strengths/weaknesses / strategy for how we are going to localize